controllers/workloads/testutils/shared_test_utils.go

```diff
@@ -13,6 +13,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	CFAppLabelKey         = "workloads.cloudfoundry.org/app-guid"
+	CFProcessGUIDLabelKey = "workloads.cloudfoundry.org/process-guid"
+	CFProcessTypeLabelKey = "workloads.cloudfoundry.org/process-type"
```

Comment:
Why do we need these constants in the tests? Looks like they are already publicly defines [here](https://github.com/cloudfoundry/cf-k8s-controllers/blob/b2a098cc334d229def7fd642be3f6ab4de05327e/apis/workloads/v1alpha1/shared_types.go#L8-L12)

Reply:
We have generally avoided using production code to test production code. Having it this way makes sure any unintentional changes to the values is caught by our tests.